src/data.json

```diff
@@ -691,7 +691,7 @@
         "atomic_number": "87",
         "media": "media/87francium.gif",
         "name": "Francium",
-        "phrase": "**Fr**ance's Eiffel Tower. Built for the World's Fair and commemorated the centenniel of the French Revolution which technically began in 17**87**.",
+        "phrase": "**Fr**ance's Eiffel Tower. Built for the World's Fair and commemorated the centennial of the French Revolution, which some might say began in 17**87**.",
```

Comment:
I can't find any argument that the French Revolution began in 1787; I'm not particularly excited by the weasely text I use in the memory peg, but at least it isn't factually incorrect. (I didn't want to make a major change—e.g., use a completely different idea for the peg—at least not in this PR.)

Reply:
I will modify this one, stating that the construction of the Eiffel tower started in 1**8**87 with no ambiguity!